pkg/rdt/rdt_test.go

```diff
@@ -219,9 +219,9 @@ kubernetes:
 
 	// Check that existing groups were read correctly on init
 	classes := GetClasses()
-	verifyGroupNames(classes, []string{"Guaranteed", "DEFAULT", "Stale"})
+	verifyGroupNames(classes, []string{"Guaranteed", "Stale", "system/default"})
```

Comment:
nitpick: is there any particular reason not to use the `RootClassName` constant here and in similar tests ?

Reply:
Good point, not really, other than legacy 😸  I think these just dated back to when there was no such constant yet defined. Changed to `RootClassName`